CMakeLists.txt

```diff
@@ -1,12 +1,9 @@
 cmake_minimum_required(VERSION 2.6)
 
 project(libbarrett)
-
-set(libbarrett_VERSION_MAJOR 1)
-set(libbarrett_VERSION_MINOR 3)
 set(libbarrett_VERSION_MAJOR 1)
 set(libbarrett_VERSION_MINOR 2)
-set(libbarrett_VERSION_PATCH 3)
+set(libbarrett_VERSION_PATCH 4)
```

Comment:
We need to update support.barrett.com/Libbarrett and the main page with the new version info, and the new installation instructions.